src/components/Icons/Icons.tsx

```diff
@@ -0,0 +1,52 @@
+// tslint:disable:max-line-length
+import * as React from "react";
+
+export const Icons = {
+
+    git: (
```

Comment:
One SVG - one file please. They must be in one folder, but without index file (loading index file will load all SVG - it wont be suitable when we will implement bundles)